tensorflow/python/ops/image_ops_impl.py

```diff
@@ -1786,6 +1885,18 @@ def convert_image_dtype(image, dtype, saturate=False, name=None):
   type, and when casting from a signed to an unsigned type; `saturate` has no
   effect on casts between floats, or on casts that increase the type's range).
 
+  Usage Example:
+    >>> x = [[[1.0, 2.0, 3.0],
+    ...       [4.0, 5.0, 6.0]],      
+    ...     [[7.0, 8.0, 9.0],      
+    ...       [10.0, 11.0, 12.0]]]
```

Comment:
Here I would construct `x` as a tensor with one `dtype` (probably `tf.int`?)

Reply:
Hmm, I don't quite understand what you are asking. Are you asking me to change them to like just "1" instead of "1.0" ?

Reply:
`x = tf.constant(..., dtype=tf.int)`